appinventor/components/src/com/google/appinventor/components/runtime/Slider.java

```diff
@@ -109,6 +130,25 @@ public Slider(ComponentContainer container) {
       Log.d(LOG_TAG, "Slider initial min, max, thumb values are: " +
           MinValue() + "/" + MaxValue() + "/" + ThumbPosition());
     }
+
+    if (DEBUG) {
+      Log.d(LOG_TAG, "API level is " + SdkLevel.getLevel());
+    }
+
+
+    // Warn the user if the API level is less than Jellybean, since getThumb
+    // does not work before API level 16.  Note that this is *not* sufficient in dealing
+    // with very old phones.   See the SeekBarHelper class above.
+    if (SdkLevel.getLevel() < SdkLevel.LEVEL_JELLYBEAN) {
```

Comment:
@halatmit I'm not sure if this is appropriate. I just tested this code on Gingerbreak (2.3.3) and I received the warning, but the slider worked as expected.